tests/tst_ncio.F90

```diff
@@ -55,6 +56,19 @@ program tst_ncio
   allocate(values_1d(5))
   values_1d = (/1,2,3,4,5/)
 
+  print *,'*** Test quantize data...'
+  nbits = 8 ! eight bit only
+  quantize1 = 314.1592653589793
+  quantize1(5,:) = 123.456789
+  quantize1(10,:) = 1013.254321
+  call quantize_data(quantize1, quantize2, nbits, qerr)
+  if (abs(quantize2(1,1)-quantize1(1,1)) .gt. qerr) then
```

Comment:
Could you explain (in comments) exactly what this test is doing?

I'm a little worried that the rest results also depend on calling quantize.

Reply:
Basically this test creates 'lossy compression' by packing/scaling the data into integers and then back to floats in order to help with zlib compression. We exercise this in GFSv16 for the 3D fields. The existing tests shouldn't depend on this I don't think.

Reply:
OK, firstly, can you put that explanation in code comments? So that it is there the next time someone looks at the code.

I'm not sure what you mean by "the existing tests shoudn't depend on this"...